fusionfrontend/src/app/fleet/components/content/asset-series-wizard/asset-series-wizard-general-information/asset-series-wizard-general-information.component.ts

```diff
@@ -57,4 +66,31 @@ export class AssetSeriesWizardGeneralInformationComponent implements OnInit {
     });
   }
 
+  onImageUpload(event: any): void {
+    this.deletePreviouslyUploadedImage();
```

Comment:
If the process is terminated, the old Image has been removed and only the default image remains. This could be problematic if uploading a new Image was done by mistake.